spec/requests/api/rest/admin/dialpeer_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::Rest::Admin::DialpeersController, type: :request do
+  include_context :json_api_admin_helpers, type: :dialpeers
+  include_context :init_routing_tag_collection
+  describe 'POST /api/rest/admin/dialpeers without gateway_id and gateway_group_id' do
```

Comment:
instead of 
describe 'POST /api/rest/admin/dialpeers without gateway_id and gateway_group_id' do
use inner block
describe 'POST /api/rest/admin/dialpeers' do
  context 'without gateway_id and gateway_group_id' do
    ...
  end
  ...
end